### ISSUE TITLE
cannot use streaming transactions in a cluster ( v3.6.1 )

### ISSUE THREAD

**Issue by mikestaub** [arangodb/arangojs#655]
Am I doing something wrong? I am getting this error on node 10.15.0 and 12.12.0:

`HPE_INVALID_HEADER_TOKEN`

Here is a quick way to reproduce:
https://github.com/mikestaub/arangojs/commit/8f4648df88b26cd46c3f5553103284310ca16618

I tried working around by using the old nodejs http parser but it is not working:
https://stackoverflow.com/questions/36628420/nodejs-request-hpe-invalid-header-token

This is currently blocking me from migrating to Oasis.

**Comment by pluma** [comment]
~~The current nodejs 12 LTS release is 12.16.1, can you try with that version?~~

I've downgraded to latest nodejs 12 LTS, spun up the cluster from the docker-compose file you've provided and adjusted the tests accordingly and ~~they're still passing.~~ am now able to reproduce this for stream transactions.

I'll investigate.

**Comment by pluma** [comment]
@joerg84 looks like ArangoDB is generating a bad response. This is the raw response ArangoDB sends:

```
HTTP/1.1 200 OK
X-Content-Type-Options: nosniff
X-Arango-Request-Forwarded-To: CRDN-e5c5319b-ffe5-4330-bc30-98f532b9db15
Content-Type: application/json
Server: ArangoDB
Http/1.1: OK
Connection: Keep-Alive
Content-Length: 77

{"code":200,"error":false,"result":{"id":"1099551670784","status":"running"}}
```

Note the stray `Http/1.1: OK` "header". I'm pretty sure that's not supposed to be there. This only seems to happen in a cluster, I couldn't reproduce this in the standalone.

**Comment by joerg84** [comment]
Thanks @pluma, looking now. @mikestaub Can you let us know whether the workaround helped?

**Comment by jsteemann** [comment]
@pluma: can you let us know what request you tried sending when you got that response? 
Then it will be so much easier to debug and less guessing needs to be done.

**Comment by pluma** [comment]
@jsteemann that particular response was from this request:

```
GET /_db/testdb_1586173041531/_api/transaction/1099551670288 HTTP/1.1
content-type: text/plain
x-arango-version: 30500
authorization: Basic cm9vdDo=
Host: localhost:8530
Connection: keep-alive
```

**Comment by mikestaub** [comment]
@pluma That workaround did not work for me, see https://github.com/mikestaub/arangojs/commit/8e640b6f1f1beb8e75d3a9c2c3de62e62f27c17f

**Comment by jsteemann** [comment]
@pluma: can you check if the problem is fixed via https://github.com/arangodb/arangodb/pull/11402? 
Thanks!

**Comment by mikestaub** [comment]
I can confirm this is resolved in arangodb 3.6.3

**Comment by joerg84** [comment]
Thanks!
